### PR TITLE
WIP: Turn on telco bastion cnf-features-deploy periodic

### PR DESCRIPTION
**Pull request by johnbieren**
Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml [hunk 1]
```diff
@@ -103,6 +103,20 @@ tests:
           cpu: 100m
           memory: 200Mi
     workflow: openshift-e2e-gcp
+- as: e2e-bastion-periodic
+  cluster: build01
+  cron: 0 0 * * *
+  steps:
+    test:
+    - as: e2e-bastion-periodic
+      commands: KUBECONFIG=$SHARED_DIR/kubeconfig make setup-test-cluster && make
```
Comment (yuvalk) [inline ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:112]: seems like you are setting KUBECONFIG only for the first make command
I think it'd make more sense to use
```
KUBECONFIG=$SHARED_DIR/kubeconfig make functests-on-ci
```

Comment (johnbieren) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:112]: good catch. I had that originally iirc, but we split it out into pieces to see at what point it was really failing. I did mess up setting the KUBECONFIG when I split it out though

**Comment by johnbieren** [comment]
Closing in favor of https://github.com/openshift/release/pull/18250

**Pull request closed without merging**